### PR TITLE
Spvc tests: Assemble explicitly for Vulkan1.1 env

### PR DESCRIPTION
**Pull request by dneto0**
By default, SPIRV-Tools will assemble for the latest supported
version of SPIR-V.  With SPIR-V 1.4 arriving, that will generally
make modules which are not consumable by Vuklan 1.1.  So update
the Spvc tests to explicitly assemble for Vulkan 1.1.

### CODE REVIEW

**Comment by dneto0** [comment]
This corresponds to the SPIRV-Cross change https://github.com/KhronosGroup/SPIRV-Cross/pull/969

**Comment by fjhenigman** [comment]
> I think the failure is acceptable, since it needs the deps roll to be fixed.

Your project now, you create revs that don't pass tests if you want.  :)

**Comment by zoddicus** [comment]
> > I think the failure is acceptable, since it needs the deps roll to be fixed.
> 
> Your project now, you create revs that don't pass tests if you want. :)

The specific failure is coming from the changing the tests in anticipation of the uprev to SPIR-V 1.4. I am blocking my other PR until that is resolved, but I think this specific PR is part of getting everything rolled ahead and into a good state, so it makes sense to land with a know failure.

**Comment by dneto0** [comment]
> Your project now, you create revs that don't pass tests if you want. :)

The trouble is there's a disconnect between SPIRV-Tools smoketest vs. Shaderc's self-test.

- SPIRV-Toosl smoke test builds Shaderc along with latest SPIRV-Tools and SPIRV-Headers.  Those latter are now defaulting to SPIR-V 1.4 and with the absorption of some SPV_GOOGLE_* extensions into core the default disassembly for certain tokens changes their string output. That's fixed by having merging #608 into Shaderc master
- Shaderc uses SPIRV-Tools and SPIRV-Headers pinned to older versions that don't support SPIR-V 1.4 yet.  This is causing the failures in #612 

Basically we sacrificed the Shaderc bots for the sake of the SPIRV-Tools bots.    This MR is part of a multi-step path to getting Shaderc back to stability, without re-breaking SPIRV-Tools bots.
